addons/stock_dropshipping/views/sale_order_views.xml

```diff
@@ -19,6 +19,15 @@
                 <xpath expr="//page/field[@name='order_line']/tree/field[@name='product_id']" position="attributes">
                     <attribute name="attrs">{'readonly': ['|', ('product_updatable', '=', False), ('purchase_line_count', '&gt;', 0)], 'required': [('display_type', '=', False)],}</attribute>
                 </xpath>
+                <xpath expr="//div[@name='button_box']" position="inside">
```

Comment:
there's already a stat button calling action_view_purchase_orders using _get_purchase_orders, no need to add this